src/TodoList.tsx

```diff
@@ -0,0 +1,21 @@
+import Todo from './types';
+
+type TodoListProps = {
+  todos: Todo[];
+};
+
+const TodoList: React.FC<TodoListProps> = ({ todos }) => {
+  return (
+    <>
+      {todos.map((todo) => {
+        return (
+          <p>
+            {todo.id} {todo.title}
```

Comment:
"todo" is not an object anymore. We can't access id and title this way.

Reply:
Done